pkgs/top-level/aliases.nix

```
@@ -1530,6 +1530,8 @@ mapAliases {
   oraclejdk11 = throw "All Oracle JDKs and JREs were dropped due to being unmaintained and heavily insecure. OpenJDK provides compatible replacements for JDKs and JREs."; # Added 2024-11-01
   OSCAR = oscar; # Added 2024-06-12
   osxfuse = throw "'osxfuse' has been renamed to/replaced by 'macfuse-stubs'"; # Converted to throw 2024-10-17
+  overrideSDK = "overrideSDK has been removed as it was a legacy compat
+ibility stub, use `apple-sdk_*` or `darwinMinVersionHook` in build inputs instead; see <https://nixos.org/manual/nixpkgs/stable/#sec-darwin> for documentation"; # Added 2025-08-04
```

String has unintentional newline.
```suggestion
  overrideSDK = "overrideSDK has been removed as it was a legacy compatibility stub. Use `apple-sdk_*` or `darwinMinVersionHook` in build inputs instead. See <https://nixos.org/manual/nixpkgs/stable/#sec-darwin> for documentation"; # Added 2025-08-04
```

---

Thanks, fixed!